### ISSUE TITLE
 Quick Compile failed. Error writing to compiler shell.

### ISSUE THREAD

**Issue by blackjyn** [BowlerHatLLC/vscode-as3mxml#439]
I'm trying to use Quick Compile but failed? 
```
(fcsh) compile 1

Quick Compile failed. Error writing to compiler shell.
```

**update: this is the screenshot from the output panel**
After the first msg appear it then displays another msg 
![as3vscode-quick-compile-err](https://user-images.githubusercontent.com/6046191/72513636-ce380000-387f-11ea-97f5-542407759bec.png)

**Comment by joshtynjala** [comment]
Unfortunately, I cannot reproduce this issue.

The next time that it happens, can you see if a stack trace has been logged? You'll need to open the **Output** view in VSCode and select **ActionScript & MXML Language Server** from the drop-down.

Which SDK are you using?

**Comment by blackjyn** [comment]
Oh I'm sorry for not giving enough info previously.

I'm using Royale SDK .

When I use normal build that is `ctrl+shift+b` it works just fine but not with quick compile

**Comment by blackjyn** [comment]
The output prints things like `java piped closed... ` like that.
I'm on travel and will re check that later

**Comment by blackjyn** [comment]
I'm updating the issue with screenshot

**Comment by joshtynjala** [comment]
Okay, thanks. It looks like it is failing to find one of Royale's required dependencies. I'll check it out.

I recommend using the regular build for now.

For what it's worth, the Quick Compile command with Royale doesn't benefit as much as other SDKs, so you're not losing much in terms of performance by using a regular build instead.

**Comment by joshtynjala** [comment]
Unfortunately, I am unable to reproduce this issue. I tried with Royale version 0.9.6 and a nightly build of 0.9.7, but with both versions, I am able to run Quick Compile & Debug repeatedly without errors.

**Comment by blackjyn** [comment]
Well, when I download and use **v.0.9.6** the **Quick Compile & Debug** works then, so I think the wiki should be updated with this additional info.

Now, after reading this:
https://github.com/BowlerHatLLC/vscode-as3mxml/wiki/Debug-a-transpiled-ActionScript-project-in-Visual-Studio-Code-with-Google-Chrome#build-automatically-before-debugging , I'm really get confused because when I initially hit **F5**  Chrome debugger launched along WITH the source being built first.

However during debug session, when I edited the source class being debugged then hit  the green "refresh button"  it just refresh the page on chrome without rebuilding the source?? 

Can you clarify that this kind of mechanism is different from that so called "Hot Reload" like in Flutter?

Or did I do something wrong? or?

I don't know whether or not this such functionalities will require to touch the Royale's source SDK to be happened. BUT will be very nice if this kind of "Hot Reload" thing comes to Royale.

**Comment by joshtynjala** [comment]
There is no hot reload in Royale at this time. You need to stop the debugger and start a new session.

Closing, since this was resolved by updating to Royale 0.9.6.

**Comment by blackjyn** [comment]
Thanks for the clarification.
